app/controllers/finders_controller.rb

```diff
@@ -90,6 +90,10 @@ def result_set_presenter_class
     ResultSetPresenter
   end
 
+  def sort_presenter
+    content_item.sorter_class.new(content_item.as_hash, filter_params)
```

Comment:
might want to memoize this one because it is called a number of times. Maybe better to instantiate in the 'show' method?